### ISSUE TITLE
[grind]

### ISSUE THREAD

**Issue by leszekjocz** [lingdb/ielex2#14]
A more trivial problem.

The proposed example sentence is:

"He is grinding flour."

My question is:
Does one grind "flour" or "seeds/grain".

**Comment by tonyakdewey** [comment]
My judgement of the English is that we can either have "He is grinding flour (from seeds/grain)" or "He is grinding seeds/grain (into flour)", i.e. this kind of belongs to the spray/load class of verbs. To eliminate this ambiguity, can we do something like "He is grinding the rocks against each other"?

**Comment by leszekjocz** [comment]
"He is grinding the rocks against each other"

I have no idea which verb I would use in this case in my mother tongue ;)
I think grinding "flour/seeds/grain" is a very good choice, my question was only about a precise wording of the example sentence.

**Comment by tonyakdewey** [comment]
Ah ha!

My one issue is that many of my languages have a specific verb for grinding seeds/grain/flour ('mill'), separate from grinding other things. Hence why I wanted to get rid of seeds/grain/flour.

**Comment by BrittaIrslinger** [comment]
Grinding seeds/grain/flour would be the most proto-typical one. In case of other materials we run the risk of getting different verbs like to wear down, to rub off, to abrade ...

**Comment by findell** [comment]
Could we a note to the definition to avoid words that _specifically_ refer to grinding foodstuffs?  That way we dodge the "mill" issue.

**Comment by leszekjocz** [comment]
I agree with Britta, we should keep the context with seeds/grain/flour. Otherwise there's a great risk of numerous synonyms.

**Comment by cormacanderson** [comment]
I'm not sure if this is really a good Swadesh term, as it's cultural vocabulary. However, it seems to be a very good term in IE - lots of branches have cognates for "mill". For this reason, in this specific case, I would be inclined to stick to the current description and example. Given the general principles though, "grind" is probably the Swadesh term in my English at least (is that true also for ye @tonyakdewey, @findell)? Outside of that usage, "grind" isn't well defined in my idiolect. As a verb I'd mainly use it in "grind away" for the noise of a machine, or the action of a person doing a tedious task. The rocks example would elicit "rub against" or something, unless defined much more tightly.

**Comment by tonyakdewey** [comment]
I use "grind" quite a bit in English, but probably because I grind my teeth in my sleep. =-)

For flour, pepper, salt, coffee, and other foodstuffs, I can say either "grind" or "mill", though the appliance/utensil used is a pepper mill, coffee mill, etc.

Can we somehow revise the description so that we can see whether we're looking for the "grind" word or the "mill" word?  Because I have both of these in most of my languages.

**Comment by cormacanderson** [comment]
Interesting, for me I would never use "mill" but only ever "grind" as a verb for the foodstuffs, and while I agree on pepper mill, I'd call it a coffee grinder :). A difference across the Atlantic @findell ? Forgot about the grinding teeth, which is actually quite a nice illustration of the prototypical motion, but not really a good example I think. It could be that this is just a Germanic problem and is straightforward enough elsewhere (maybe even a rare case of entering two lexemes for this meaning in some Germanic languages?), but we can certainly revise the description to try to tighten this up. How would you suggest doing that?

**Comment by findell** [comment]
My usage is much the same as @cormacanderson's - I would say either pepper grinder or pepper mill, but I would grind pepper with it (illogical as that is).  Grinding teeth seems to me less prototypical - I think of grinding seeds as the most basic example.  Not sure if that solves anything.
